.github/workflows/integration.yml

```diff
@@ -38,4 +38,4 @@ jobs:
           SENTRY_ESLINT_RELAXED: 1
         run: |
           cd sentry
-          yarn lint
```

Comment:
biome kept running and throwing errors? we don't inherit the biome lint rules though